libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/persistence/Utils.java

```diff
@@ -0,0 +1,42 @@
+package com.linkedin.openhouse.datalayout.persistence;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import com.linkedin.openhouse.datalayout.layoutselection.DataLayoutOptimizationStrategy;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.spark.sql.SparkSession;
+
+/** Utility class for strategies persistence. */
+public final class Utils {
+  public static final String DATA_LAYOUT_PROPERTY_KEY = "data-layout-strategies";
```

Comment:
Should we follow format "compaction.data-layout-strategies", if this property is specific to compaction processes?
Easier to identify with grouping.

Reply:
thanks, compaction is too specific, we can make it consistent with iceberg naming convention, like write.data-layout.strategies.